src/resources/js/tickets.js

```diff
@@ -438,8 +452,11 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 					$tribe_tickets.trigger( 'deleted-ticket.tribe', response );
 
 					if ( response.success ) {
-						$tribe_tickets.trigger( 'clear.tribe' );
-						$( 'td.ticket_list_container' ).empty().html( response.data );
+						// remove deleted ticket from table
+						var $deleted_row = $( '#tribe_ticket_list_table' ).find( '[data-ticket-order-id="order_' + deleted_ticket_id + '"]' );
+						$deleted_row.remove();
```

Comment:
line break between this and variable declaration above it 

would also encourage using `document.getElementById` instead of `$( '#tribe_ticket_list_table' )`